hackathon/models.py

```diff
@@ -153,6 +158,20 @@ class HackProject(models.Model):
         related_name="hackproject",)
     display_name = models.CharField(default="", max_length=255)
     description = models.TextField(max_length=500)
+    technologies_used = models.CharField(
```

Comment:
Another "nice to have", but for a Hackathon itself, is this necessary?

Reply:
Again yeah, nice to have. I don't think I made it required.